lib/instance.js

```diff
@@ -383,6 +383,13 @@ module.exports = (function() {
             }
           }
 
+          // parse the well known text
+          if (this.Model._hasGeometryAttributes && this.Model._isGeometryAttribute(key) && value !== null && value !== undefined && !value._isSequelizeMethod) {
```

Comment:
We don't need the ` && value !== null && value !== undefined && !value._isSequelizeMethod` part, when we check `isString` right after :)